proposals/0126-atf-additional-transports.md

```diff
@@ -33,27 +33,28 @@ Main reasons :
 ## Proposed solution
 
 Use real mobile device as a transport adapter.
-Create Mobile application as a part of ATF infrastructure. 
-Mobile application may use [SDL android library](https://github.com/smartdevicelink/sdl_android)
-for communicating with SDL and TCP connection for communication with ATF.
-
-Mobile application (Mobile transport adapter) should provide to ATF side such RPCs:
- - ConnectToSDL(transport)
- - RemoveConnection(connection_id)
- - GetListOfAvailableTransports()
- - SendData(connection_id, data)
- 
-ATF should be able to connect to multiple Mobile transport adapters and provide ability to use any of them to test engineer. 
-
-ATF should provide such API to test engineer: 
- - ConnectToSDL(device, connection type)
- - RemoveConnection(connection)
- - SendRawData(connection)
- - GetListOfAvailableDevices()
- - GetDeviceInfo(device)
- 
-_СonnectToSDL_ should return connection object.
-Connection object should provide ability to create sessions on it. Session interface should not be changed.  
+Create Android mobile application as a part of ATF infrastructure.
+
+Mobile application should use HTTP connection with REST protocol for control communication with ATF using next API:
+ - GetDeviceInfo - Provides device's information
+ - ConnectToSDL - Connects to SDL and provides proxy TCP connection parameters
+ - DisconnectFromSDL - Disconnects from SDL
+ - GetSDLConnectionStatus - Provides status of connection to SDL
+
+Mobile application should provide next information as main part of response to GetDeviceInfo request:
+ - Operating system
+ - MAC address
+ - Array of transports available to connect to SDL
```

Comment:
@aderiabin suggest describing in parenthesis what are the elements of the array are. Is that just a transport name or another structure? What should it contain?

Reply:
@AKalinich-Luxoft I have added such information. (see ae31abb)
But I think these are details of implementation and further adding of details to API lead to full API specification. I would like to leave it not specified in proposal.

Reply:
@AKalinich-Luxoft Information related to implementation details has been removed. (see 36b2cdb)